Formula/solidity.rb

```diff
@@ -3,7 +3,7 @@ class Solidity < Formula
   homepage "https://solidity.readthedocs.io"
   url "https://github.com/ethereum/solidity/releases/download/v0.7.1/solidity_0.7.1.tar.gz"
   sha256 "c69205d902ac8dd66f5aeaa78a08e5411cdb26a0b9479fcf44f394b7008b484c"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

Isn't it `-or-later`?

```c++
/*
	This file is part of solidity.

	solidity is free software: you can redistribute it and/or modify
	it under the terms of the GNU General Public License as published by
	the Free Software Foundation, either version 3 of the License, or
	(at your option) any later version.

	solidity is distributed in the hope that it will be useful,
	but WITHOUT ANY WARRANTY; without even the implied warranty of
	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
	GNU General Public License for more details.

	You should have received a copy of the GNU General Public License
	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
*/
```

I'm seeing that header a lot (e.g. [`solidityUpgrade/main.cpp`](https://github.com/ethereum/solidity/blob/905065280f8de24865cec8c7b990b0ef1f5b305f/tools/solidityUpgrade/main.cpp#L1-L16))

Also, from the [`README`](https://github.com/ethereum/solidity#license):

> Some third-party code has its [own licensing terms](https://github.com/ethereum/solidity/blob/develop/cmake/templates/license.h.in).

If we need to include these as well, it looks like it should be:

```suggestion
  license all_of: ["GPL-3.0-or-later", "MIT", "BSD-3-Clause", "Apache-2.0", "CC0-1.0"]
```

Reply:
Thanks @Rylan12! You're right, it's `GPL-3.0-or-later`. But I'm not sure with `all_of`, I will check and update.